zerver/lib/events.py

```diff
@@ -254,6 +255,9 @@ def fetch_initial_state_data(user_profile: UserProfile,
             state[notification] = getattr(user_profile, notification)
         state['default_desktop_notifications'] = user_profile.default_desktop_notifications
 
+    if want('support_email'):
+        state['support_email'] = FromAddress.SUPPORT
```

Comment:
I think we should just make this the settings.ZULIP_ADMINISTRATOR or whatever it is. The FromAddress.SUPPORT will be something like "Zulip Support" <email>.